lambdas/utils/cloudwatch_logs_query.py

```diff
@@ -43,10 +43,24 @@ class CloudwatchLogsQueryParams:
     """,
 )
 
+LloydGeorgeRecordsSearched = CloudwatchLogsQueryParams(
+    lambda_name="SearchPatientDetailsLambda",
+    query_string="""
+        fields @timestamp, Message, Authorisation.selected_organisation.org_ods_code AS ods_code
+        | filter Message = 'Searched for patient details' 
+        | stats count() AS daily_count_searched BY ods_code
+    """,
+)
+
+
 UniqueActiveUserIds = CloudwatchLogsQueryParams(
-    lambda_name="AuthoriserLambda",
+    lambda_name="TokenRequestHandler",
     query_string="""
-        fields @timestamp, Authorisation.selected_organisation.org_ods_code AS ods_code, Authorisation.nhs_user_id AS user_id
+        fields @timestamp,
+        Authorisation.selected_organisation.org_ods_code AS ods_code,
+        Authorisation.nhs_user_id AS user_id,
+        Authorisation.selected_organisation.role_code AS role_code,
+        Authorisation.repository_role AS user_role
         | filter ispresent(ods_code) AND ispresent(user_id)
         | dedup(ods_code, user_id)
```

Comment:
```suggestion
        | dedup(ods_code, user_id, user_role)
```

tested in ndr-test and it was only counting one role per login, Noga suggested trying this and it resolved the issue so multiple roles are counted